### PR TITLE
Configuration for decoration blocks

### LINKED ISSUE #1443: witchwood biome/tree spawn rate
would it be possible to have the spawn rate for both witchwood biome and witchwood trees configurable in AM2 config this would help with balence for other mod and sever usage

thanks 
johnfg12

### PR DESCRIPTION
**Pull request by TheIcyOne**
Resolves #1443, adds configuration options for Witchwood, Etherium Pool and Wakebloom generation rate, as well as number of attempts to generate flowers.

Also added description to a few config options as they were previously missing.

### CODE REVIEW

**Comment by TheIcyOne** [comment]
Ignore the long list of commits, the last one is the only one that matters. I totally remembered to use branches.

**Comment by jjtParadox** [comment]
It would be preferable if you could clean up the commit list, as we'd like to try to keep our commit history somewhat coherent. Look into branching and rebasing, as those may help.

**Comment by TheIcyOne** [comment]
Thanks for the help, I was wondering how to do that :+1:

**Comment by jjtParadox** [comment]
@TheIcyOne are those oregen config changes supposed to be a part of this PR? If not, you may want to move them to another PR, as you don't want code sneaking into the wrong places :stuck_out_tongue_winking_eye:

**Comment by TheIcyOne** [comment]
Yeah, I felt oregen fits in quite well with the rest of the changes in the PR.

File: src/main/java/am2/worldgen/AM2WorldDecorator.java [hunk 1]
```diff
@@ -18,6 +18,8 @@
 import java.util.ArrayList;
 import java.util.Random;
 
+import javax.security.auth.callback.ConfirmationCallback;
```
Comment (jjtParadox) [inline src/main/java/am2/worldgen/AM2WorldDecorator.java]: These two lines should probably be removed.

Comment (TheIcyOne) [inline src/main/java/am2/worldgen/AM2WorldDecorator.java]: Did... did I add those? o.O

**Pull request merged**